tests/rkt_export_test.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/coreos/rkt/common"
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func TestExport(t *testing.T) {
+	const (
+		testAci     = "test.aci"
+		testFile    = "test.txt"
+		testContent = "ThisIsATest"
+		runInspect  = "%s %s %s %s --exec=/inspect -- %s"
```

Comment:
I am not convinced by the factoring with this constant runInspect: it is declared far away from where it is used and I cannot easily check what each '%s' is linked to.